test/ClangImporter/cxx_interop.swift

```diff
@@ -1,5 +1,5 @@
 // RUN: %empty-directory(%t)
-// RUN: %target-swift-frontend -typecheck %s -I %S/Inputs/custom-modules -module-cache-path %t -enable-cxx-interop
+// RUN: %target-swift-frontend -emit-ir -o %t %s -I %S/Inputs/custom-modules -module-cache-path %t -enable-cxx-interop -verify
```

Comment:
Can you make this a separate test? I think we'll probably want to diagnose things about imported types in this main test.